crates/egui-wgpu/src/renderer.rs

```diff
@@ -299,10 +290,23 @@ impl Renderer {
             multiview: None,
         });
 
+        const VERTEX_BUFFER_START_CAPACITY: wgpu::BufferAddress =
+            std::mem::size_of::<Vertex>() as wgpu::BufferAddress * 1024;
+        const INDEX_BUFFER_START_CAPACITY: wgpu::BufferAddress =
+            std::mem::size_of::<u32>() as wgpu::BufferAddress * 1024 * 3;
```

Comment:
trick for slightly less code:

```suggestion
        const VERTEX_BUFFER_START_CAPACITY: wgpu::BufferAddress =
            (std::mem::size_of::<Vertex>() * 1024) as _;
        const INDEX_BUFFER_START_CAPACITY: wgpu::BufferAddress =
            (std::mem::size_of::<u32>() * 1024 * 3) as _;
```